compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/internal/parser/BallerinaParser.java

```diff
@@ -1333,16 +1333,11 @@ private STNode parseParamGivenAnnots(SyntaxKind prevParamKind, STNode annots, bo
         STNode qualifier;
         STToken nextToken = peek();
         switch (nextToken.kind) {
-            case PUBLIC_KEYWORD:
-                qualifier = parseQualifier();
-                break;
             case IDENTIFIER_TOKEN:
-                qualifier = STNodeFactory.createEmptyNode();
                 break;
             case AT_TOKEN: // Annotations can't reach here
             default:
                 if (isTypeStartingToken(nextToken.kind) && nextToken.kind != SyntaxKind.IDENTIFIER_TOKEN) {
-                    qualifier = STNodeFactory.createEmptyNode();
                     break;
                 }
```

Comment:
Since we don't parse any qualifiers here, this method simply does nothing. 
1. Can remove this entire method and direct call `parseParamGivenAnnotsAndQualifier()` beneath. 
2. rename: `parseParamGivenAnnotsAndQualifier` -> `parseParamGivenAnnots`